.npmrc

```diff
@@ -1,2 +1,3 @@
-engine-strict=true
+# Temporarily disabled because of dependency mixups in mongosh 0.6.1.
+#engine-strict=true
```

Comment:
Opened https://jira.mongodb.org/browse/COMPASS-4535 to make sure we revert this after the next mongosh update, see https://github.com/mongodb-js/mongosh/pull/487